pros/cli/common.py

```diff
@@ -194,28 +194,32 @@ def pros_root(f):
     return decorator
 
 
-def resolve_v5_port(port: Optional[str], type: str, quiet: bool = False) -> Optional[str]:
+def resolve_v5_port(port: Optional[str], type: str, quiet: bool = False) -> Tuple[Optional[str], bool]:
```

Comment:
There are a lot of uses of this in v5_utils.py that aren't updated. I don't have a brain on me to double check, but I assume this will break those.

Reply:
Can you also add comment on what the different parts of the tuple are? That way you don't have to read the function if you want to use it.